package.json

```diff
@@ -11,7 +11,8 @@
     "prepublishOnly": "npm run build",
     "bundle": "cd tools/bundler && npm i && npm run bundle",
     "startNewVersion": "newVersion=$npm_config_new_version node scripts/add-new-version.js",
-    "lint": "echo 'No linter integrated yet'",
+    "lint": "npx eslint . --ext .js",
```

Comment:
You can borrow this line from Parser, for example: https://github.com/asyncapi/parser-js/blob/master/package.json#L36

Also feel free (optional) to add the `lint:fix` lines as well.

Reply:
> No, it wasn't. And I think it will be better to remove it from this PR and rather add it later on. The only requirement I would say we should ask for in any PR is to, at least, a basic linting by eslint on each code pushed. Even though running it locally.
_Originally posted by @smoya in https://github.com/asyncapi/spec-json-schemas/pull/452#discussion_r1402729934_

Do we need this here after [your recent comment](https://github.com/asyncapi/spec-json-schemas/pull/452#discussion_r1402729934) about including the JavaScript linting?

Reply:
You can remove all linting scripts. But make sure you run a minimum lint to your code before last push ok?

Reply:
> You can remove all linting scripts. But make sure you run a minimum lint to your code before last push ok?

something like this :point_down: 
```json
"lint-schemas": "npx eslint ./scripts/validate-schemas-final.js && node scripts/validate-schemas-final.js"
```

Reply:
> Also feel free (optional) to add the lint:fix lines as well.

should I add this now to `package.json`?

Reply:
No. Do not add anything about linting in package.json. Do not include the eslint dependency either. 
Just run the linter locally every time you do push your code so you ensure your code is linted.

Reply:
>  Just run the linter locally every time you do push your code so you ensure your code is linted.

push as in?

Reply:
> No. Do not add anything about linting in package.json. Do not include the eslint dependency either

understood.

Reply:
marking resolved as we don't have to include linting in this PR, we will do it separately in a new issue.